index.d.ts

```diff
@@ -320,6 +707,27 @@ declare namespace AutoNumeric {
 
     export type SerializeSpacesOption = "+" | "%20";
 
+    export type OptionsHandler = {
+        [K in keyof Options]-?: (value: Exclude<Options[K], undefined>) => AutoNumeric
+    } & {
+       /**
+        * Reset any options set previously, by overwriting them with the default settings
+        * @returns This auto numeric instance for chaining method calls.
```

Comment:
Please change `auto numeric` to `AutoNumeric`

Reply:
There were also various different ways how it was spelled. Ok, so `AutoNumeric` is the preferred way, I'll update it.